extensions/ui/org.eclipse.smarthome.ui.basic/ESH-INF/config/config.xml

```diff
@@ -5,6 +5,18 @@
         http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
 	<config-description uri="ui:basic">
+		<parameter name="theme" type="text" required="true">
```

Comment:
This is now a pretty different approach than we have in the Paper UI.
With the Paper UI, we allow ESH solutions to add a custom theme on top by adding a fragment. It is NOT a feature exposed to the user, though. I think this makes sense in terms of ensuring a "corporate identity", i.e. allowing solutions to make sure that their UIs carry their logo and match there general CI.

While I can understand that it can be useful to allow users to choose a light or dark theme, I am not so convinced, if we really need orange/blue/green color options (resp. rather allow solutions to add additional themes besides light/dark through a fragment).

Reply:
I'll leave just Default and Dark then. We can work on the fragment solution later on if that's preferred.

Reply:
Done